common/app/conf/switches/ABTestSwitches.scala

```diff
@@ -22,27 +22,7 @@ trait ABTestSwitches {
     "This places the epic on all articles for all users, with a limit of 4 impressions in any given 30 days",
```

Comment:
I know this repo is coming to an end but having this permanent business logic as a 'test' is odd to me